metron-interface/metron-alerts/src/app/pcap/pcap-panel/pcap-panel.component.html

```diff
@@ -1,7 +1,6 @@
-<button (click)="test()">Test</button>
-<div class="panel">
-  <div classs="panel-header">
-    <app-pcap-filters (search)="onSearch($event)"></app-pcap-filters>
+<div class="panel my-4">
+  <div class="panel-header">
+    <app-pcap-filters (search)="test()"></app-pcap-filters>
```

Comment:
any reason to code in the test here?

Reply:
I am assuming we will switch it back to the real call at some point.  Just wanted to give you a preview of how the panel with look and behave.  Happy to change it back now or leave this PR open and add to it until we get the backend integration done.

Reply:
fair enough. we will have to take out my dodgy hack test button anyway.